Cesium.Compiler/Cesium.Compiler.csproj

```diff
@@ -6,14 +6,29 @@ SPDX-License-Identifier: MIT
 
 <Project Sdk="Microsoft.NET.Sdk">
 
+    <PropertyGroup>
+        <PackageLicenseExpression>
+            Apache-2.0 <!-- Yoakke.* --> AND $(PackageLicenseExpression) AND MS-PL <!-- QuickGraph, QuickGraph.Graphviz -->
+        </PackageLicenseExpression>
+        <Copyright>
+            © Microsoft Corporation. All rights reserved. <!-- Microsoft.Bcl.HashCode, Microsoft.Bcl.Memory -->
+            Copyright (c) 2005 - 2015 Giacomo Stelluti Scala &amp; Contributors; <!-- CommandLineParser -->
+            Copyright (c) 2016-2025 JetBrains s.r.o. <!-- JetBrains.Annotations -->
+            Copyright 2019 Bradley Grainger; <!-- IndexRange -->
+            Copyright © 2019, 2020 Alexandre Rabérin; <!-- QuickGraph, QuikGraph.Graphviz -->
+            Copyright (c) 2021 LanguageDev team; <!-- Yoakke.* -->
+            2024-2025 TruePath contributors &lt;https://github.com/ForNeVeR/TruePath&gt;; <!-- TruePath -->
+            $(Copyright)
```

Comment:
Should we move our copyright first, and not MS, that can give wrong impression when read not carefully.

Reply:
I sorted these in chronological order, but I guess you are right and we can put ourselves first.